### PR TITLE
Feature/landing page

### PR DESCRIPTION
**Pull request by MeganHuggins**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Styling- no new features
- [ ] Test

#### What's this PR do(Detailed Discription)?

* This PR goes over the initial set up needed for our React Nativ app which will be utilizing both Redux as the global state management, as well as React Navigation for our routing. The initial react components that have been created are that of App, LandingPage, NavBar, Form, Map and ActiveTrip. So far the Landing page has been hooked up to the React Navigation and has begun to be styled out. Also, I began the set up for our Redux store and components individual actions and reducers (This is not complete however)

#### Where should the reviewer start?

* Start first at the App.js and view how the React Navigation is set up and will work for the project. From there, inspect the LandingPage.js to see the basic set up occurring in there and if any questions arise from this. 

#### How should this be manually tested?

* I have set up testing files for each of the components which will be utilized a bit further down the road. We will need to set up these tests using the logic to access the Redux store when we get to these.

#### Any background context you want to provide?

* I am still trying to read through the docs of how exactly to set up Redux with React Nativ but I found a good site going over it which I'll link below: 
[Redux](https://www.reactnative.guide/9-redux/9.1-redux-setup.html)

#### Questions:

* Should we create a different file (index.js) to hold our redux wrapper and import App.js into there? I think that might keep things a bit cleaner

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,21 +1,40 @@
...
+const Stack = createStackNavigator();
 
 export default function App() {
   return (
-    <View style={styles.container}>
-      <Text>Open up App.js to start working on your app!</Text>
-      <StatusBar style="auto" />
-    </View>
+    <NavigationContainer>
+      <Stack.Navigator initialRouteName='Landing'>
```
Comment (infamouskeyduster) [inline App.js:16]: This stack navigator looks promising. I'm interested to learn more about it, to see if it is going to suit our needs!

File: src/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            colors={['rgba(196, 196, 196, 0.66)', 'rgba(196, 196, 196, 0)']}
+            style={styles.linearGradient}
+          >
+            <Text style={styles.logoHeader}>
+              {'Hifu'.toUpperCase()}
+            </Text>
+            <Text style={styles.logoText}>
+              {'When you want to get lost but still found'.toUpperCase()}
+            </Text>
+            <TouchableOpacity
```
Comment (infamouskeyduster) [inline src/LandingPage/LandingPage.js:30]: Touchable opacity — interested to see what this is all about.

File: App.js (hunk 1 above)
Comment (MeganHuggins) [reply App.js:16]: From what I've read and from other peoples experience when it comes to react native and routing this is the best/ most intuitive way to go with it. Basically, it's just set up on the app.js and is the only place that the routing needs to be from my understanding.

File: src/LandingPage/LandingPage.js (hunk 2 above)
Comment (MeganHuggins) [reply src/LandingPage/LandingPage.js:30]: Yeah! It's a super cool part of react native that you can use/ import for the buttons that you want to be able to be clickable/ interactive for you site. Here are the docs for it: 
[Touchable](https://reactnative.dev/docs/touchableopacity)

**Review by infamouskeyduster — APPROVED** [review]
Great job getting our components set up & adding some really interesting built in React Native functionality / components.

**Pull request merged**